### ISSUE TITLE
there should be hyphen support for romaji

### ISSUE THREAD

**Issue by ImUrX** [WaniKani/WanaKana#142]
Before opening a new issue, please take a moment to review our [**community guidelines**](https://github.com/WaniKani/WanaKana/blob/master/.github/CONTRIBUTING.md) to make the contribution process easy and effective for everyone involved.

**Before opening a new issue, you may find an answer in already closed issues**:
https://github.com/WaniKani/WanaKana/issues?q=is%3Aissue+is%3Aclosed

## Issue Type

* [ ] Bug (https://github.com/WaniKani/WanaKana/blob/master/.github/CONTRIBUTING.md#bug-reports)
* [x] Feature (https://github.com/WaniKani/WanaKana/blob/master/.github/CONTRIBUTING.md#feature-requests)

## Description
So when for example writing ``recoodo`` it should also support ``reco-do`` for writing ``レコード``

**Comment by DJTB** [comment]
Thanks for your comment, though I'm not sure I understand your request:
```javascript
wanakana.toKana('RECO-DO')
'レコード'
wanakana.toKatakana('reco-do')
'レコード'
```

Likewise, with the IME we convert `-` to `ー` if the user enters a hyphen.

If you're actually asking that `recoodo` could automatically convert to `reco-do`.  I don't think we will consider support for converting extended vowel sounds to a long dash in katakana the way we already do with hiragana. It's normal for the long dash to be explicitly entered via a hyphen rather than repeated characters.

**Comment by ImUrX** [comment]
yeah you are right, using ``toKatakana`` is just the way to do it lol. There are very few cases of - being used for that in hiragana
